src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -123,6 +138,45 @@ public void Load(string settingsXml)
 #endif
         }
 
+        public void SaveRandomSeed(string dirname)
+        {
+            TextWriter writer = null;
+
+            try
+            {
+                writer = new StreamWriter(Path.Combine(dirname, "$RANDOM_SEED$"));
+                writer.Write(RandomSeed.Value);
```

Comment:
It's only for local use, so I think it's fine. We can revisit if we end up saving any more stuff to pass between discovery and execution.